rosdep/python.yaml

```
@@ -7044,6 +7044,13 @@ python3-pydantic:
       pip: [pydantic]
     xenial:
       pip: [pydantic]
+python3-pydbus:
+  debian: [python3-pydbus]
+  fedora: [python3-pydbus]
+  opensuse: [python3-pydbus]
+  rhel:
+    '8': [python3-pydbus]
```

For future proofing, I think we could write:
```suggestion
    '*': [python3-pydbus]
    '7': null
```

@wjwwood Can you double-check?

---

Looking at other rhel examples I found that they use `python%{python3_pkgversion}-` and that lead me down the path to find that for rhel7 this package is available but with the name `python36-pydbus`. I think that setting this as `python%{python3_pkgversion}-pydbus` for all versions might make this work.

---

It seems that wasn't actually the case. python36-pydbus doesn't actually exist for RHEL 7. I've updated it to the commended null for 7.